home:
  x: -1.86
  y: -0.61
  
Table1:
  x: -0.35
  y: 1.73

Table2:
  x: 0.11
  y: -1.93

Table3:
  x: 1.81
  y: 0.61

Table4:
  x: 0.54
  y: 0.51
  
Table5:
  x: -1.84
  y: 0.29
  
home2:
  x: 0.01
  y: -2.19
  

